src/modules/jobs/controllers/pokemon.controller.ts

```diff
@@ -0,0 +1,35 @@
+import { Controller, Get, Query } from '@nestjs/common';
+
+import { PokemonService } from '../services/pokemon.service';
+
+@Controller('/pokemon')
+export class PokemonController {
+  constructor(private readonly pokemonService: PokemonService) {}
+
+  /**
+   * 
```

Comment:
can you add a @description here to explain what this route does. You aren't going to ever call this method elsewhere in the nest app, but it's good practice so that if someone references a method they can see a desc for what it does